### PR TITLE
Further export command tests

### PR DESCRIPTION
**Pull request by dwaynebailey**
Gets us close to 100% for export command

### CODE REVIEW

File: tests/commands/export.py [hunk 1]
```diff
@@ -31,9 +32,52 @@ def test_export_project(capfd, afrikaans_tutorial, french_tutorial):
 
 @pytest.mark.cmd
 @pytest.mark.django_db
+def test_export_project_and_lang(capfd, afrikaans_tutorial, french_tutorial):
```
Comment (phlax) [inline tests/commands/export.py]: no need to use these fixtures use ``project0`` and ``language0``

File: tests/commands/export.py [hunk 2]
```diff
@@ -31,9 +32,52 @@ def test_export_project(capfd, afrikaans_tutorial, french_tutorial):
...
     """Export a language"""
     call_command('export', '--language=af')
     out, err = capfd.readouterr()
     assert 'af.zip' in out
     assert 'fr.zip' not in out
+
+
+@pytest.mark.cmd
+@pytest.mark.django_db
+def test_export_path(capfd, afrikaans_tutorial, french_tutorial):
```
Comment (phlax) [inline tests/commands/export.py]: same as above

**Comment by unho** [comment]
lgtm, but mind comments by others.

File: tests/commands/export.py (hunk 1 above)
Comment (dwaynebailey) [inline tests/commands/export.py]: @phlax can you give me a quick lesson then, I'm not sure I understand the new fixtures

Comment (phlax) [inline tests/commands/export.py]: yep np

**Pull request merged**